### DISCUSSION TITLE
Question about assemblies

### DISCUSSION THREAD

**Discussion by kingofthebongo2008** [nxrighthere/UnrealCLR#37] [General]
Hey, 
I am testing the plugin.
I have noticed that if I add another assembly as a reference to my .net core app 3.1 my app is not loaded in unreal.
It turns out that it loads the first assembly in the directory.

My assembly name was UnrealEngine.SmartMirrorDYU.dll
I had NewtonSoft.Json.dll and it did not load,
When i changed the name to AAAMirror.dll it start to work again.
Is there an entry point ?

**Comment by nxrighthere** [comment]
Your assemblies should be a part of a single project that referenced with `UnrealEngine.Framework.dll`, or loaded as plugins using assembly load contexts in loaded assembly.

**Comment by nxrighthere** [comment]
@kingofthebongo2008 There was a bug with assemblies inspection logic which should be fixed now. Naming should not affect loading anymore.

**Comment by kingofthebongo2008** [comment]
All good now.
Thank you
